src/index.js

```diff
@@ -134,8 +138,12 @@ class Dropzone extends React.Component {
   }
 
   open() {
+    const clickEvent = new MouseEvent('click', {
+      bubbles: false
+    });
+
     this.fileInputEl.value = null;
-    this.fileInputEl.click();
+    this.fileInputEl.dispatchEvent(clickEvent);
```

Comment:
Unfortunately not (that's what I tried first), the onClick that happens there is a synthetic React event so `e.stopPropagation` doesn't work on it. See: http://stackoverflow.com/a/24421834/1661462